src/me/StevenLawson/TotalFreedomMod/Command_owner.java

```diff
@@ -0,0 +1,41 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import java.util.ArrayList;
+import java.util.List;
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
+import me.StevenLawson.TotalFreedomMod.TFM_PlayerRank;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.ALL, source = SourceType.BOTH)
+@CommandParameters(description = "Checks if the owner is online or offline.", usage = "/<command> [-a | -i]", aliases = "who")
+public class Command_owner extends TFM_Command
+
+    @Override
+    public boolean run(CommandSender sender, Command cmd, String label, String[] args) {
+           Player slime = Bukkit.getServer().getPlayer("markbyron");
```

Comment:
Why have you assigned "slime" to the player? Thats just confusing, and to be honest makes it look like you ripped that code from something else pre-made. This should really be player or something similar.

The next thing, what if Mark changes his name? Your snookered then. This should really get the player by UUID if possible.